sycl/test/syclcompat/launch/launch_policy_neg.cpp

```diff
@@ -148,7 +147,7 @@ void test_variadic_config_ctor() {
     sycl::nd_range<3> launch_range{{1, 1, 32}, {1, 1, 32}};
     compat_exp::local_mem_size lmem_size(0);
     compat_exp::launch_policy duplicate_local_mem_config(launch_range, lmem_size, lmem_size);
-    //CHECK8: error: static assertion failed due to requirement{{.*(exactly once|duplicate type)}}
```

Comment:
Won't this break on linux now as it is? I'd have thought you need to have different checks depending on OS if they use different error messages?

Probably I'm missing something. It seems that you only have to change the message for CHECK8 to fix the failure? So that the error message is only different on windows for this case?

Reply:
This is the error message with GCC 12:
```
error: static assertion failed due to requirement '__idx < sizeof...(_Types)': the type T in std::get<T> must occur exactly once in the tuple
```
So the check still applies. The other checks are not affected.

Reply:
oh yes, I see.